### PR TITLE
Final changes

### PR DESCRIPTION
**Pull request by advaypal**
1.  Cleanup code.
2. Fix storage test by creating separate config file for tests. This means changing the structure of the config file slightly.
3. Jump to relevant task on updating, and jump to end of the list on add.

### CODE REVIEW

**Comment by Skaty** [comment]
@advaypal Please resolve conflicts.

**Comment by advaypal** [comment]
@Skaty Resolved

File: src/main/java/seedu/task/model/tag/UniqueTagList.java [hunk 1]
```diff
@@ -29,7 +34,12 @@ protected DuplicateTagException() {
     /**
      * Signals that an operation tried to operate on a tag that does not exist
      */
-    public static class TagNotFoundException extends Exception { }
+    public static class TagNotFoundException extends Exception {
+
+        /**
+         *
```
Comment (Skaty) [inline src/main/java/seedu/task/model/tag/UniqueTagList.java]: empty comment block for?

**Review by Skaty — CHANGES_REQUESTED** [review]
RFC

File: src/main/java/seedu/task/model/tag/UniqueTagList.java (hunk 1 above)
Comment (advaypal) [reply src/main/java/seedu/task/model/tag/UniqueTagList.java]: Got added when adding the stupid serial number. Will change and push

Comment (Skaty) [reply src/main/java/seedu/task/model/tag/UniqueTagList.java]: change soon.

**Comment by advaypal** [comment]
@Skaty changes made

**Review by Skaty — APPROVED** [review]

**Pull request merged**